test/policies/category_policy_test.exs

```diff
@@ -0,0 +1,29 @@
+defmodule CodeCorps.CategoryPolicyTest do
+  use CodeCorps.PolicyCase
+
+  import CodeCorps.CategoryPolicy, only: [create?: 1, update?: 1]
+
+  describe "create?" do
+    test "returns true when user is an admin" do
+      user = build(:user, admin: true)
+      assert create?(user) == true
+    end
+
+    test "returns false if user is not an admin" do
+      user = build(:user, admin: false)
+      assert create?(user) == false
```

Comment:
There seem to be a few tests that explicitly check against true or false. I think we could leverage `assert` versus `refute`